packages/cli-tool/src/add-ons/ui-framework/tailwind-css.ts

```diff
@@ -0,0 +1,100 @@
+import * as fs from 'fs';
+
+import { cli } from 'cli-ux';
+
+import runCommand from '../../helpers/child-process';
+import { replaceLine } from '../../helpers/file-editor';
+
+const NPM_PACKAGES = [
```

Comment:
In my opinion, those packages alongside bootstrap's ones should be placed somewhere in the root folder like tooling-version or at-least cli-tool with package.json. It could be helped when the options are extended or just simply bumping to new version

Reply:
As it is a template, it would always install the **latest** minor version (using `^`). Regarding major version, we probably don't want an-auto bumping. 
The problem I see with putting everything in the root is that it can become quite messy if later we start adding more and more add-ons... I tend to prefer having "modular" add-ons — i.e. self-contained.

Let me know what you think. 
If you believe it is very important, feel free to add a new issue, as an improvement for later ;-)

Reply:
@malparty I meant when you put tailwindcss lib list and others like boostrap's ones in separated folders when you need to bump them manually, you have to go direct to every file of them to adjust instead of modifying directly on the shared file. But I agree it's not a considerable problem.

Reply:
One more small wander, you install those packages with function by `npm install` script, so those libs are recorded on package-lock but package.json. What if they remove node_modules and package-lock or forget to add them in the dockerfile, it could take time to investigate to find out the problem.